### DISCUSSION TITLE
Enhanced DAG Navigation with Search Bar on Dag View Page

### DISCUSSION THREAD

**Discussion by sshah90** [apache/airflow#33205] [Ideas] [answered]
As a data engineer, I frequently find myself hopping between different DAGs. However, the current process involves navigating back to the Airflow home page each time I need to switch to a new DAG.

To streamline this experience and improve overall efficiency, I would like to propose the addition of a search bar placed at the top of the DAG level page(the search bar is similar to the attached image).

<img width="1792" alt="Screenshot 2023-08-07 at 1 59 27 PM" src="https://github.com/apache/airflow/assets/26821376/be98c6ad-8bb6-4ad3-9b47-5b2f23ff57ea">

**Note**: I am not UX/UI developer so please forgive my styling on the image.

**Comment by potiuk** [comment, marked as answer]
Yes. Why not. You can create a feature request for it - hopefully someone picks it up and implement it

**Reply by sshah90** [reply]
Thank you @potiuk 

let me open feature request.
